keyboards/ergodox_stm32/i2cmaster.c

```diff
@@ -0,0 +1,101 @@
+/* Copyright 2018 Jack Humbert
```

Comment:
Please remove this file. 

Specifically, we want to use `/drivers/arm/i2c_master.c` instead of a custom implementation.

Reply:
I guess I should modify /driver/ implementation to work with STM32 F1 devices then?

Reply:
In theory, it should already.

Reply:
In fact, it looks almost identical to the existing implementation, except for some hard coded behavior. 

So you really shouldn't need to do much here, at all.

Reply:
Oh so the chibiOS API has 2 different initialization structures for F1 and F4 etc.. i2C the initlization part needs to change. And that’s all. Both the GPIO and i2C config struct. I am thinking about doing ifdefs for that. Detect F1 family device.